pkg/plugins/policies/imports.go

```diff
@@ -1,18 +1,60 @@
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshaccesslog"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshcircuitbreaker"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshfaultinjection"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshhealthcheck"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshloadbalancingstrategy"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshmetric"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshproxypatch"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshratelimit"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshretry"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtcproute"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtimeout"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtrace"
-	_ "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission"
+	"os"
+	"strings"
+
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshaccesslog"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshcircuitbreaker"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshfaultinjection"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshhealthcheck"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshloadbalancingstrategy"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshmetric"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshproxypatch"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshratelimit"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshretry"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtcproute"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtimeout"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtrace"
+	"github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission"
 )
+
+var nameToModule = map[string]func(){
+	"meshaccesslogs":              meshaccesslog.InitPlugin,
+	"meshcircuitbreakers":         meshcircuitbreaker.InitPlugin,
+	"meshfaultinjections":         meshfaultinjection.InitPlugin,
+	"meshhealthchecks":            meshhealthcheck.InitPlugin,
+	"meshhttproutes":              meshhttproute.InitPlugin,
+	"meshloadbalancingstrategies": meshloadbalancingstrategy.InitPlugin,
+	"meshmetrics":                 meshmetric.InitPlugin,
+	"meshproxypatches":            meshproxypatch.InitPlugin,
+	"meshratelimits":              meshratelimit.InitPlugin,
+	"meshretries":                 meshretry.InitPlugin,
+	"meshtcproutes":               meshtcproute.InitPlugin,
+	"meshtimeouts":                meshtimeout.InitPlugin,
+	"meshtraces":                  meshtrace.InitPlugin,
+	"meshtrafficpermissions":      meshtrafficpermission.InitPlugin,
+}
+
+func initAllPolicies() {
+	for _, initializer := range nameToModule {
+		initializer()
+	}
+}
+
+func init() {
+	// we read this manually otherwise we would have to wire in enabling plugins in every test
+	rawEnabledPluginPolicies := os.Getenv("KUMA_PLUGIN_POLICIES_ENABLED")
```

Comment:
What is unfortunate about this approach is that this env var is not documented and it's going with "different flow" than the regular config. Can't we include this in our config and just load policies in bootstrap phase?

You said that this might be a problem for tests. If so, we could maybe init policies as part of `RunSpecs` which is called for every test

Reply:
I'll try that approach

Reply:
@jakubdyszkiewicz - please look at https://github.com/kumahq/kuma/pull/8828